File: server/app/static/js/photo.js
// <-----------------------------------------< Header >----------------------------------------->
//
//       photo.py
//       By: Fredrick Stakem
//       Date: 3.7.14
//
//
// <-----------------------------------------<---~~--->----------------------------------------->


// Libraries
// None

function Photo(name, path)
{
	this.name = name;
	this.path = path;
};

Photo.prototype = 
{
	equals: function(other_photo)
	{
		if(this.name == other_photo.name && this.path == other_photo.path)
		{
			return true;
		}

		return false;
	}
};

File: server/app/photo.py
# <-----------------------------------------< Header >----------------------------------------->
#
#       photo.py
#       By: Fredrick Stakem
#       Date: 3.13.15
#
#
# <-----------------------------------------<---~~--->----------------------------------------->

"""
This file is used to hold information about a camera photo.

"""


# Libraries
#


class Photo(object):

	def __init__(self, name, path, url):
		self.name = name
		self.path = path
		self.url = url

	def __str__(self):
		output = 'Photo: %s @ %s' % (self.name, self.short_path)
		return output